powerfactory_tools/interface.py

```diff
@@ -1495,7 +1661,10 @@ def create_study_case(
         if scenario is not None:
             self.activate_scenario(scenario)
 
-        self.switch_study_case(current_study_case.loc_name)
+        if current_study_case is not None:
+            self.switch_study_case(current_study_case.loc_name)
+        else:
+            self.deactivate_study_case(study_case)
```

Comment:
why deactivate?

Reply:
The goal is to create and not to also activate the new study case.
--> We need to temporaly activate the newly created study case to activate grids, scenarios and variants if specified by the user. After this setup, we switch back to the formerly activated study case or just deactivate the study case created.